### ISSUE TITLE
[USH-914] The purpose of the "Add new post" button is not reflected in its Angular component name

### ISSUE THREAD

**Issue by Ifycode** [ushahidi/platform-client-mzima#622]
#### Issue description

The purpose of the "Add new post" button is not reflected in way the component is named in the code.

#### How to reproduce or check for this issue

* Follow the instructions to launch the web client locally at: [https://github.com/ushahidi/platform-client-mzima#platform-client-mzima](https://github.com/ushahidi/platform-client-mzima#platform-client-mzima).
* The "Add new post" is in the sidebar on the user interface, this should give you an idea of where to find it in the code.

#### Suggested/expected output

Rename the Angular component to `add-web-post-button` instead (without breaking the functionality).

<sub>From [SyncLinear.com](https://synclinear.com) | [USH-914](https://linear.app/ushahidi/issue/USH-914/the-purpose-of-the-add-new-post-button-is-not-reflected-in-its-angular)</sub>

**Comment by shakirandagire** [comment]
@Ifycode Could you assign me to this issue.

**Comment by Ifycode** [comment]
One issue at a time @shakirandagire as stated here in the **hacktoberfest** discussion: https://github.com/ushahidi/platform/discussions/4650

You can always come back to ask to be assigned to this ticket once your other issue ticket's submission is approved. That is, if no one else has requested for this issue ticket (and assigned to it) before you get back here again.

**Comment by shakirandagire** [comment]
Alright thank you @Ifycode. Will come back to this after the other PR is merged.

**Comment by Ifycode** [comment]
Your PR doesn't have to be merged (Sometimes merging could take days, a week or more depending on what we want to achieve). 

As long as your PR is approved you can always pick another issue and start work on it @shakirandagire 

 Cc: @MugoBrian @oyenuga17

**Comment by Ifycode** [comment]
Your PR doesn't have to be merged (Sometimes merging could take days, a week or more depending on what we want to achieve). 

As long as your PR is approved you can always pick another issue to get assigned to @shakirandagire 

 Cc: @MugoBrian @oyenuga17

**Comment by shakirandagire** [comment]
@Ifycode Thanks for the information

**Comment by shakirandagire** [comment]
@Ifycode could you assign me to this issue

**Comment by Ifycode** [comment]
Assigned to you @shakirandagire 
Thank you.
